tools/load_demo_data.py

```diff
@@ -34,80 +38,168 @@
+
+                data = assert_post('instrument',
+                                   data={'name': 'P60 Camera',
+                                         'type': 'phot',
+                                         'band': 'optical',
+                                         'telescope_id': telescope1_id})
+                instrument1_id = data['data']['id']
+
+                data = assert_post('telescope',
+                                            data={'name': 'Nordic Optical Telescope',
+                                                  'nickname': 'NOT',
+                                                  'lat': 28.75,
+                                                  'lon': 17.88,
+                                                  'elevation': 1870,
+                                                  'diameter': 2.56,
+                                                  'group_ids': [group_id]
+                                            })
+                telescope2_id = data['data']['id']
+
+                data = assert_post('instrument',
+                                   data={'name': 'ALFOSC',
+                                         'type': 'both',
+                                         'band': 'optical',
+                                         'telescope_id': telescope2_id
+                                   })
+
+            with status("Creating dummy sources"):
+                SOURCES = [{'id': '14gqr', 'ra': 353.36647, 'dec': 33.646149,
+                            'redshift': 0.063, 'group_ids': [group_id],
+                            'comments': ["No source at transient location to R>26 in LRIS imaging",
+                                         "Strong calcium lines have emerged."]},
+                           {'id': '16fil', 'ra': 322.718872, 'dec': 27.574113,
+                            'redshift': 0.0, 'group_ids': [group_id],
+                            'comments': ["Frogs in the pond", "The eagle has landed"]}]
+
+                (basedir/'static/thumbnails').mkdir(parents=True, exist_ok=True)
+                for source_info in SOURCES:
+                    comments = source_info.pop('comments')
+
+                    data = assert_post('sources', data=source_info)
+                    assert data['data']['id'] == source_info['id']
+
+                    for comment in comments:
+                        data = assert_post('comment',
+                                           data={'source_id': source_info['id'],
+                                                 'text': comment})
+
+                    phot_file = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                                             'skyportal', 'tests', 'data',
+                                             'phot.csv')
+                    phot_data = pd.read_csv(phot_file)
+
+                    data = assert_post('photometry',
+                                       data={'source_id': source_info['id'],
```

Comment:
I'm actually quite curious what [black](https://pypi.org/project/black/) would recommend in this case.